NGitLab.Mock/Clients/MergeRequestClient.cs

```diff
@@ -262,6 +263,27 @@ public Models.MergeRequest Close(int mergeRequestIid)
 
                 mergeRequest.ClosedAt = DateTimeOffset.UtcNow;
                 mergeRequest.UpdatedAt = DateTimeOffset.UtcNow;
+
+                var currentUser = Context.User;
+                Server.ResourceStateEvents.Add(new ResourceStateEvent()
```

Comment:
A little method to avoid duplication? You'll just need to passe the state, user & merquerequestid